### PR TITLE
feat: expose content-bounds-updated event

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
This exposes a new WebContents event, `content-bounds-updated`, which is called
when the page calls `window.moveTo` or `window.resizeTo`. It is possible to
prevent the default behavior of moving the window by calling
`event.preventDefault()`.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] tests are [changed or added](https://github.com/electron/electron/blob/main/docs/development/testing.md)
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Added new WebContents event `content-bounds-updated`.

### CODE REVIEW

**Review by ckerr — COMMENTED** [review]
API LGTM.

Is it possible to add a test for this?

File: docs/api/web-contents.md [hunk 1]
```diff
@@ -130,10 +130,6 @@ Corresponds to the points in time when the spinner of the tab stopped spinning.
 
 #### Event: 'dom-ready'
 
-Returns:
-
-* `event` Event
-
```
Comment (ckerr) [inline docs/api/web-contents.md:136]: Is this removal intentional?

It creates this artifact diff:

```
-    on(event: 'dom-ready', listener: (event: Event) => void): this;
-    once(event: 'dom-ready', listener: (event: Event) => void): this;
-    addListener(event: 'dom-ready', listener: (event: Event) => void): this;
-    removeListener(event: 'dom-ready', listener: (event: Event) => void): this;
+    on(event: 'dom-ready', listener: Function): this;
+    once(event: 'dom-ready', listener: Function): this;
+    addListener(event: 'dom-ready', listener: Function): this;
+    removeListener(event: 'dom-ready', listener: Function): this;
```

...which is maybe a good change since AFAICT there's nothing the user can do with `event`, but as far as docs go it does lose the type info about the return type.

Comment (nornagon) [reply docs/api/web-contents.md:136]: The return type doesn't matter either. Arguably that's a bug in the ts generator but I don't think it's worth worrying about.

This removal is intentional (though unrelated to the main change). I think the artifact change is a good one.

The Event object will still be passed, though as you noticed it's completely useless, so I think it's correct to drop it from the docs.

**Comment by nornagon** [comment]
tests added.

**Review by codebytere — APPROVED** [review]
API LGTM
